src/System.Data.Common/src/System/Data/XMLSchema.cs

```diff
@@ -783,7 +783,6 @@ public void LoadSchema(XmlSchemaSet schemaSet, DataSet ds)
                     continue;
                 }
 
-                DataTable table = HandleTable(element);
```

Comment:
Why is it ok to delete this?  Doesn't HandleTable have side effects?

Reply:
@stephentoub 

After looking at the logic inside `HandleTable`, you're right. I think it may contain side effect caused by the logic within HandleTable and one of the side effect is having throw Exception, 

Although the logic within HandleTable itself only read the property of the passed `element` parameter, not changing the parameter value at all. 

I think `HandleTable` doesn't change the state/value of the passed `element`, can we keep this line that calls `HandleTable` to be deleted? After all, the returned `table` is not used anywhere else.

Reply:
> can we keep this line that calls HandleTable to be deleted?

I'm not confident it's safe change.  In fact I suspect it's related to the CI failures.

Please put it back.  Thanks.

Reply:
@stephentoub 
Done putting back the line and using discard to ignore return value.

Thanks for the patience! 🙂